README.md

```diff
@@ -168,6 +167,15 @@ snakemake -j 1
 
 ![default workflow](./docs/basic_workflow.png)
 
+You will want to pass snakemake the specific GraphKB instance you are working with as well as the credentials of the user that will be uploading. If you have followed the docker install demo instructions this might looks something like this
+
+```bash
+snakemake -j 1 \
+  --config gkb_user='graphkb_importer' \
+  gkb_pass='secret' \
+  gkb_url='http://localhost:8080/api'
+```
+
 The COSMIC and DrugBank options require licensing and are therefore not run by default. If you have
 a license to use them then you can include one or both of them by providing email and password
 as config parameters
```

Comment:
it would be snakefile -h to check the config parameter names?

Reply:
i don't know that there is a way to do that actually. snakemake -h would just give you the snakemake help menu